packages/@aws-cdk/aws-route53resolver-alpha/test/integ.firewall.ts

```diff
@@ -1,46 +1,48 @@
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
 import * as path from 'path';
 import { Vpc } from 'aws-cdk-lib/aws-ec2';
-import { App, Stack, StackProps } from 'aws-cdk-lib';
-import { Construct } from 'constructs';
+import { App, Stack } from 'aws-cdk-lib';
 import * as route53resolver from '../lib';
 
-class TestStack extends Stack {
-  constructor(scope: Construct, id: string, props?: StackProps) {
-    super(scope, id, props);
-
-    const vpc = new Vpc(this, 'Vpc', { maxAzs: 1, restrictDefaultSecurityGroup: false });
-
-    const blockList = new route53resolver.FirewallDomainList(this, 'BlockList', {
-      domains: route53resolver.FirewallDomains.fromList(['bad-domain.com', 'bot-domain.net']),
-    });
-    const overrideList = new route53resolver.FirewallDomainList(this, 'OverrideList', {
-      domains: route53resolver.FirewallDomains.fromList(['override-domain.com']),
-    });
-
-    new route53resolver.FirewallDomainList(this, 'OtherList', {
-      domains: route53resolver.FirewallDomains.fromAsset(path.join(__dirname, 'domains.txt')),
-    });
-
-    const ruleGroup = new route53resolver.FirewallRuleGroup(this, 'RuleGroup');
-
-    ruleGroup.addRule({
-      priority: 10,
-      firewallDomainList: blockList,
-      action: route53resolver.FirewallRuleAction.block(),
-    });
-    ruleGroup.addRule({
-      priority: 20,
-      firewallDomainList: overrideList,
-      action: route53resolver.FirewallRuleAction.block(route53resolver.DnsBlockResponse.override('amazon.com')),
-    });
-
-    ruleGroup.associate('Association', {
-      priority: 101,
-      vpc,
-    });
-  }
-}
-
 const app = new App();
-new TestStack(app, 'cdk-route53-resolver-firewall');
-app.synth();
+const stack = new Stack(app, 'cdk-route53-resolver-firewall');
+const vpc = new Vpc(stack, 'Vpc', { maxAzs: 1, restrictDefaultSecurityGroup: false });
+
+const blockList = new route53resolver.FirewallDomainList(stack, 'BlockList', {
+  domains: route53resolver.FirewallDomains.fromList(['bad-domain.com', 'bot-domain.net']),
+});
+const overrideList = new route53resolver.FirewallDomainList(stack, 'OverrideList', {
+  domains: route53resolver.FirewallDomains.fromList(['override-domain.com']),
+});
+// const managedMalwareList = new route53resolver.FirewallManagedDomainList(stack, 'ManagedMalwareList', {
+//   managedDomainList: route53resolver.ManagedDomain.MALWARE_DOMAIN_LIST,
+// });
```

Comment:
Help wanted:
When I remove the comment out of these lines and try to update the snapshot, custom resource creation fails with the following error:
> Received response status [FAILED] from custom resource. Message returned: Error: connect ECONNREFUSED 127.0.0.1:443 at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1495:16) (RequestId: a5f8ab1e-f81c-41af-b887-650d4417bcf7)

I guess there are something wrong with my implementation of custom resource handler so CloudFormation cannot receive the correct callback, but I have no idea.